Cargo.toml

```diff
@@ -182,6 +183,7 @@ inherits = "release"
 codegen-units = 16
 lto = false
 incremental = true
+debug = 1
```

Comment:
I think that the `quick-bench` below provides this profile, i.e., `quick-release`+`debug=1`, would you be okay to leave this out in favour of that?

Reply:
oh yeah no problem I meant to take this out